docs/examples/testing/unit_test.py

```diff
@@ -16,15 +18,9 @@ def inches_to_cm(self, inches):
     def cms_to_inches(self, cms):
         return self.math.divide(cms, 2.54)
 
-# =============================================================================
-# Begin test
-# =============================================================================
-
 
 def test_conversion_service():
-    # create instance
-    # dependencies are replaced with mocks unless otherwise given - see
-    # :class:`nameko.testing.services.worker_factory`
+    # create worker with mock dependencies
```

Comment:
probably worth a comment/link to http://www.voidspace.org.uk/python/mock/mock.html#mock.Mock.side_effect